keyboards/bear_face/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "keyboard_name": "bear_face", 
+  "url": "https://github.com/chemicalwill/bear_face_pcb", 
+  "maintainer": "chemicalwill", 
+  "width": 16, 
+  "height": 28, 
+  "layouts": {
+      "LAYOUT": {
```

Comment:
```suggestion
      "LAYOUT_83_ansi": {
```

Reply:
So I had removed `LAYOUTS = 83_ansi` from `rules.mk` but I'm guessing this serves a different purpose? I don't know a ton about what's going on under the hood here but would love to learn